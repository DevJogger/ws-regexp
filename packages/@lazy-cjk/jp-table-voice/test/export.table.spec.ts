import _m0 = require('../');

test(`import check`, () =>
{

	let actual = Object.keys(_m0);

	expect(actual).toMatchSnapshot();

});

test(`import default check`, () =>
{

	expect(_m0.default).toStrictEqual(_m0.getVoiceAll);

});

test(`EnumTableVoice`, () =>
{

	// @ts-ignore
	expect(_m0.EnumTableVoice).toMatchSnapshot();

});
