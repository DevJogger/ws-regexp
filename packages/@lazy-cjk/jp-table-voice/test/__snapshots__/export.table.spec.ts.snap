// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnumTableVoice 1`] = `
Object {
  "0": "01",
  "01": 0,
  "02": 1,
  "03": 2,
  "1": "02",
  "2": "03",
}
`;

exports[`import check 1`] = `
Array [
  "table_voice",
  "EnumTableVoice",
  "_buildTablePlus",
  "_get",
  "getVoiceAll",
  "toVoice01",
  "toVoice02",
  "toVoice03",
  "default",
]
`;
